src/vopr_hub/SETUP.md

```diff
@@ -1,5 +1,13 @@
-# VOPR Hub Setup
-* Follow these instructions to setup the VOPR Hub, along with several full-time VOPR simulators, on a dedicated testing machine.*
+# VOPR Server Setup
+
+*The VOPR Server is a dedicated testing machine. It continually runs VOPR simulators (performing deterministic fuzz testing) as well as another program, called the VOPR Hub (that listens out for bug reports). When a VOPR finds a bug, it creates a report and sends it to the VOPR Hub. When a report comes in the hub has an additional, dedicated VOPR to use to rerun the test. The hub then automatically creates a GitHub issue with the collected debug logs and stack trace.*
+
+*To ensure the latest code is always being tested, the VOPRs restart after running only a few seeds. Before restarting, the scheduler is responsible for fetching the latest code and checking out the relevant commit for that VOPR. At least one VOPR will always run on main. Other VOPRs can be assigned to test pull requests with the `vopr` label. The latest pull requests are always favored if there are too many to test.*
```

Comment:
Did the label name change?

Reply:
@sentientwaffle Issues with a `seed` label are created by the VOPR, and PRs that you want the VOPR to test need the `vopr` label. It hasn't changed. I must have made a mistake in the README.md file previously.

Reply:
Oh, that makes sense; I didn't realize there were two labels. Thanks!